### PR TITLE
Remove Linux images

### PR DESCRIPTION
**Pull request by nickgerace**
https://github.com/rancher/rancher/issues/33403

### CODE REVIEW

**Review by paynejacob — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -8,7 +8,6 @@ ifeq ($(shell uname), Darwin)
 endif
 
 all:
-	-rm $(MAKEPATH)/Dockerfile.windows
-	curl $(DOCKERFILE_SOURCE) -o $(MAKEPATH)/Dockerfile.windows
-	$(SED) -i 's/^ARG FLUENTBIT_VERSION.*/ARG FLUENTBIT_VERSION=$(VERSION)/g' $(MAKEPATH)/Dockerfile.windows
-	$(SED) -i 's/FLUENT_BIT_VERSION=.*/FLUENT_BIT_VERSION=$(VERSION)"/g' $(MAKEPATH)/.drone.yml
```
Comment (aiyengar2) [inline Makefile:14]: Why was this removed?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
-# Packaging for fluent-bit
+# Packaging for Windows `fluent-bit` Images
 
 [![dockerhub](https://img.shields.io/docker/v/rancher/fluent-bit?sort=semver&style=flat-square)](https://hub.docker.com/r/rancher/fluent-bit/tags)
 
-`fluent-bit` image packaging for Rancher's Logging v2, which uses the [banzaicloud/logging-operator](https://github.com/banzaicloud/logging-operator).
+This repository houses `fluent-bit` Windows image packaging for Rancher's Logging v2, which uses the [banzaicloud/logging-operator](https://github.com/banzaicloud/logging-operator).
```
Comment (aiyengar2) [inline README.md]:
```suggestion
This repository houses `fluent-bit` Windows image packaging for Rancher's Logging V2, which is based on [banzaicloud/logging-operator](https://github.com/banzaicloud/logging-operator).
```

**Review by aiyengar2 — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (nickgerace) [reply Makefile:14]: The last line was only in use for Linux images.

**Review by aiyengar2 — APPROVED** [review]
LGTM
